MusicTaste/MusicTaste/APIManagers/SpotifyAPIManager.m

```diff
@@ -9,4 +9,6 @@
 
 @implementation SpotifyAPIManager
 
+
+
```

Comment:
Try to pay attention to such changes in GitKraken. Do you still need these extra lines? If not, discard the changes before committing